### DISCUSSION TITLE
Camera orientation priors

### DISCUSSION THREAD

**Discussion by jkaufman** [colmap/colmap#2447] [Q&A]
I have approximate camera orientations for each image in a world coordinate space, but no position data. Would it be possible to use orientation priors, alone? It seems someone else [asked the inverse question](https://github.com/colmap/colmap/issues/1983#issue-1776653821), but did not receive a reply yet.

**Comment by tmnku** [comment]
What do you want to use these priors for? Colmap only allows you to use prior poses for matching images. You could use your priors to compute the list of match pairs and pass it to the matches importer.

**Reply by jkaufman** [reply]
Yes, I was hoping to use them to improve matching images with few features. Thank you for the suggestion.
